calico/operations/ebpf/enabling-ebpf.mdx

```diff
@@ -363,16 +363,23 @@ To revert to standard Linux networking:
 
 1. (Depending on whether you installed Calico with the operator or with a manifest) reverse the changes to the operator's `Installation` or the `FelixConfiguration` resource:
 
+<Tabs>
+<TabItem label="Operator" value="Operator-4">
+
```

Comment:
Intent to col 4 to line up with "(Depending ..."

This helps with the layout, so it's clearly part of the step.

Reply:
Ah OK @ctauchen - I just did simple copy n' paste similar to the existing Tab and TabItem elements for consistency which were flush with the LHS but will Indent and line up as requested